frontend/src/components/Participants.vue

```diff
@@ -0,0 +1,288 @@
+      >
+    </v-flex>
+    <v-card>
+      <v-toolbar flat>
+        <v-icon class="pa-3">fas fa-user-plus</v-icon>
+        <v-toolbar-title>Create new participant</v-toolbar-title>
+      </v-toolbar>
+      <v-divider></v-divider>
+      <v-form @keyup.enter="onSubmit" class="pa-6" ref="form">
+        <v-text-field
+          type="text"
+          label="E-mail"
+          v-model.trim="form.email"
+          @blur="$v.form.email.$touch()"
+          :error-messages="emailErrors"
+        />
+        <v-text-field
+          label="Github link"
+          v-model="form.github"
+          @blur="$v.form.github.$touch()"
+          :error-messages="githubErrors"
+        />
+        <v-text-field
+          label="First name"
+          v-model="form.first_name"
+          @blur="$v.form.first_name.$touch()"
+          :error-messages="firstNameErrors"
+        />
+        <v-text-field
+          label="Last name"
+          v-model="form.last_name"
+          @blur="validateLastName($event)"
+          :error-messages="lastNameErrors"
+        />
+        <v-text-field
+          label="Slack nick"
+          v-model="form.slack"
+          @blur="validateSlack($event)"
+          :error-messages="slackErrors"
+        />
+        <v-text-field
+          label="Phone No."
+          v-model="form.phone"
+          :counter="9"
+          @blur="validatePhone($event)"
+          :error-messages="phoneErrors"
+        />
+        <v-card-actions class="pt-3 align-center justify-center">
+          <v-btn
+            :disabled="$v.$invalid"
+            @click="onSubmit"
+            class="add-participant-btn align-center justify-center"
+            >Add new participant</v-btn
+          >
+        </v-card-actions>
+      </v-form>
+    </v-card>
+  </v-container>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { mapGetters } from 'vuex';
+import {
+  required,
+  minLength,
+  maxLength,
+  email,
+  numeric
+} from 'vuelidate/lib/validators';
+export default {
+  data() {
+    return {
+      form: {
+        email: '',
+        github: '',
+        first_name: '',
+        last_name: '',
+        slack: '',
+        phone: ''
+      },
+      warningAlert: false,
+      warningMessage: '',
+      successAlert: false
+    };
+  },
+  created() {
+    this.$store.dispatch('participant/getParticipants');
+    setInterval(() => {
+      this.successAlert = false;
+    }, 10000);
+    setInterval(() => {
+      this.warningAlert = false;
+    }, 30000);
+  },
+  validations: {
+    form: {
+      email: {
+        required,
+        email,
+        emailExists(value) {
+          return !Object.values(this.allParticipants).find(
+            user => user.email === value
+          );
+        }
+      },
+      github: {
+        required,
+        githubExists(value) {
+          return !Object.values(this.allParticipants).find(
+            user => user.github === value
+          );
```

Comment:
Why we store github urls instead of github username? Right now I am not able to add https:/
/github.com/RozaliaJasica (that github exists in db), but I can add RozaliaJasica. So if we want to keep urls, it could be worth to somehow validate it, or provide some template, to make that field persistent. Or You could just prepopulate that field with 'https://github.com/'

Reply:
Yes I know, but I believe serlizer doesn't accept urls anymore, does it? I remember getting rejects in response when I used url validation.

Reply:
I'm guessing that url could be too long - serializer validates that field with max length 38 chars. So maybe we could just replace input label from github link to e.g. github username or account ?

Reply:
Ok. The issue was that RegExp the way as it was written didn't accept signs like `/`. 

At the same time after discussing with @stanislawK we agreed that there is no reason to store URLs rather than bland usernames so I leave it as it is. 

I also added a method, which deletes "https://github.com/" automatically in case someone would want to copy/paste URL from github